lib/ews/soap/builders/ews_builder.rb

```diff
@@ -46,8 +46,9 @@ def initialize
     #       b.folder_shape!({:base_shape => 'Default'})
     #     end
     #   end
-    def build!(opts = {}, &block)
-      @nbuild.Envelope(NAMESPACES) do |node|
+    def build!(opts, &block)
```

Comment:
not sure if it's intension to make the `opts` mandatory parameter?

Reply:
Yeah, it wasn't intentional 🙈 Accidentally changed it